pageserver/src/repository/rocksdb.rs

```diff
@@ -127,20 +127,23 @@ impl CacheKey {
 static LAST_VALID_LSN_KEY: CacheKey = CacheKey::special(0);
 static LAST_VALID_RECORD_LSN_KEY: CacheKey = CacheKey::special(1);
 
+#[derive(Debug)]
 enum CacheEntryContent {
     PageImage(Bytes),
     WALRecord(WALRecord),
     Truncation,
+    Drop,
 }
 
 // The serialized representation of a CacheEntryContent begins with
 // single byte that indicates what kind of entry it is. There is also
 // an UNUSED_VERSION_FLAG that is not represented in the CacheEntryContent
 // at all, you must peek into the first byte of the serialized representation
 // to read it.
-const CONTENT_PAGE_IMAGE: u8 = 1u8;
-const CONTENT_WAL_RECORD: u8 = 2u8;
-const CONTENT_TRUNCATION: u8 = 3u8;
+const CONTENT_PAGE_IMAGE: u8 = 0u8;
+const CONTENT_WAL_RECORD: u8 = 1u8;
+const CONTENT_TRUNCATION: u8 = 2u8;
+const CONTENT_DROP: u8 = 3u8;
```

Comment:
We should figure out how to make this an `enum`.  There are a few crates out there that can help derive things like `From<u8>` or `TryFrom<u8>`: [`num_enum`](https://docs.rs/num_enum/0.5.1/num_enum/), for example.